StreamChatClient_v3/APIClient/Endpoints/MemberPayload_Tests.swift

```diff
@@ -5,11 +5,8 @@
 @testable import StreamChatClient_v3
 import XCTest
 
-class MemberEndpointPayload_Tests: XCTestCase {
-    let memberJSON: Data = {
-        let url = Bundle(for: MemberEndpointPayload_Tests.self).url(forResource: "Member", withExtension: "json")!
-        return try! Data(contentsOf: url)
-    }()
+final class MemberEndpointPayload_Tests: XCTestCase {
```

Comment:
Sorry, but I think adding `final` to test classes is just unnecessary noise 🤷‍♂️

Reply:
I don't think it's a noise. It's a good practice to think about that constantly and do the same in SDK code.